spec/recipes/minimize_access_spec.rb

```diff
@@ -27,56 +27,62 @@
     stub_command("find /bin  -perm -go+w -type f | wc -l | egrep '^0$'").and_return(false)
   end
 
-  let(:chef_run) do
+  cached(:chef_run) do
     ChefSpec::ServerRunner.new.converge(described_recipe)
   end
 
+  subject { chef_run }
+
   it 'remove write permission from /usr/local/sbin' do
-    expect(chef_run).to run_execute('remove write permission from /usr/local/sbin').with(
+    is_expected.to run_execute(
+      'remove write permission from /usr/local/sbin'
+    ).with(
       command: 'chmod go-w -R /usr/local/sbin'
     )
   end
 
   it 'remove write permission from /usr/local/bin' do
-    expect(chef_run).to run_execute('remove write permission from /usr/local/bin').with(
+    is_expected.to run_execute(
+      'remove write permission from /usr/local/bin'
+    ).with(
       command: 'chmod go-w -R /usr/local/bin'
     )
   end
 
   it 'remove write permission from /usr/sbin' do
-    expect(chef_run).to run_execute('remove write permission from /usr/sbin').with(
+    is_expected.to run_execute('remove write permission from /usr/sbin').with(
```

Comment:
Ok, lets stay with 80 and enabling of LineLength cop

Last time I tried cookstyle - it was in the very early stage, I do not know the current state of it. I think its a good idea to check this way, but a dedicated PR would be better for that from my view (in order to get this PR merged as soon as possible)